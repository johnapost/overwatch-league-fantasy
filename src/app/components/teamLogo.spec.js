import React from "react";
import { shallow } from "enzyme";
import TeamLogo from "./teamLogo";

const props = {
  team: "BOS"
};

describe("TeamLogo", () => {
  it("should render img tag with default props", () => {
    const wrapper = shallow(<TeamLogo {...props} />);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it("should render img tag with passed props", () => {
    const wrapper = shallow(<TeamLogo {...props} height="16" width="16" />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
