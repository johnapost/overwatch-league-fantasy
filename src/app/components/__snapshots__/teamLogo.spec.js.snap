// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamLogo should render img tag with default props 1`] = `"<img src=\\"static/BOS.svg\\" alt=\\"BOS\\" height=\\"32\\" width=\\"32\\"/>"`;

exports[`TeamLogo should render img tag with passed props 1`] = `"<img src=\\"static/BOS.svg\\" alt=\\"BOS\\" height=\\"16\\" width=\\"16\\"/>"`;
